runtime/oti/j9trace.h

```diff
@@ -63,6 +63,7 @@ typedef struct RasGlobalStorage {
 	void *  traceMethodTable;
 	int     stackdepth;
 	unsigned int    stackCompressionLevel;
+	unsigned int    methodStrArgLength;
```

Comment:
Suggest this new field should be named `maxStringLength` (for consistency with the keyword).